### PR TITLE
Use new pipe render system

### PR DESCRIPTION
**Pull request by brachy84**
Make laser pipes use new render system.
Also fixed a TOP issue with this. I
grafik

### CODE REVIEW

**Comment by htmlcsjs** [comment]
thanks for doing this

File: src/main/java/net/htmlcsjs/htmlTech/common/laserpipe/LaserPipeRenderer.java [hunk 1]
```diff
@@ -1,267 +1,52 @@
...
+    private final Map<LaserPipeType, TextureAtlasSprite> pipeTextures = new HashMap();
 
-    public static void preInit() {
-        HtmlTech.logger.info("initializing laser pipe renderer");
-        BLOCK_RENDER_TYPE = BlockRenderingRegistry.createRenderType("ht_laser_pipe");
-        BlockRenderingRegistry.registerRenderer(BLOCK_RENDER_TYPE, INSTANCE);
-        MinecraftForge.EVENT_BUS.register(INSTANCE);
-        TextureUtils.addIconRegister(INSTANCE::registerIcons);
+    public LaserPipeRenderer() {
+        super("ht_laser_pipe", new ResourceLocation(HtmlTech.MODID, "laser_pipe_normal"));
```
Comment (htmlcsjs) [inline src/main/java/net/htmlcsjs/htmlTech/common/laserpipe/LaserPipeRenderer.java]: Could you move this to `HTTextures.java`?

**Review by htmlcsjs — CHANGES_REQUESTED** [review]

File: src/main/java/net/htmlcsjs/htmlTech/common/laserpipe/LaserPipeRenderer.java (hunk 1 above)
Comment (brachy84) [reply src/main/java/net/htmlcsjs/htmlTech/common/laserpipe/LaserPipeRenderer.java:26]: wdym? What exactly should be put there?

Comment (brachy84) [reply src/main/java/net/htmlcsjs/htmlTech/common/laserpipe/LaserPipeRenderer.java]: Wdym? what exactly should be put there

Comment (htmlcsjs) [reply src/main/java/net/htmlcsjs/htmlTech/common/laserpipe/LaserPipeRenderer.java]: I ment, can you move the `resourceLocation` to `HTTextures.java`

**Review by htmlcsjs — APPROVED** [review]

**Pull request merged**